src/common/Voucher.js

```diff
@@ -156,6 +160,9 @@ Voucher.prototype.toArray = function() {
     if(this.complaint && this.complaint.id) {
       arr.complaintid = this.complaint.id;
     }
+    if(this.financialentity && this.financialentity.id){
```

Comment:
could you add a space before the { on this line please, eslint is throwing an error
![image](https://user-images.githubusercontent.com/94699827/172931768-e4aa91d8-04fe-4695-abcd-00dd6e7d0319.png)